util/dHomogTrans.m

```diff
@@ -32,5 +32,5 @@
 ret(5:7, :) = Rz * qdot_to_omega_x - Rx * qdot_to_omega_z;
 ret(9:11, :) = -Ry * qdot_to_omega_x + Rx * qdot_to_omega_y;
 ret(13:15, :) = R * qdot_to_twist(4:6, :);
-
+ret(16, :) = 1;
```

Comment:
This is wrong. dHomogTrans returns the gradient of a homogeneous transform. The 16th element in a homogeneous transform is the constant 1. The gradient of a constant w.r.t. anything is zero, not one.